oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/query/ElasticSuggestIterator.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.apache.jackrabbit.oak.plugins.index.elastic.query;
 
+import org.apache.jackrabbit.oak.plugins.index.elastic.ElasticIndexNode;
```

Comment:
Same as the last one - looks like an unused import.

Reply:
`ElasticIndexNode` has been moved to the parent package (since is needed by both the query and index parts) so the import is necessary.